plugins/nvme/fu-nvme-device.c

```diff
@@ -80,18 +80,10 @@ static gchar *
 fu_nvme_device_get_guid_safe (const guint8 *buf, guint16 addr_start)
 {
 	efi_guid_t guid_tmp;
-	guint guint_sum = 0;
 	g_autofree char *guid = NULL;
 
-	/* check the GUID is plausible */
-	for (guint i = 0; i < 16; i++)
-		guint_sum += buf[addr_start + i];
-	if (guint_sum == 0x00)
+	if (!fu_common_guid_get_plausible (buf, addr_start))
```

Comment:
Could we do `buf + addr_start` or is there a good reason to send the offset?

Reply:
I agree, not a good reason to send both.